src/MongoDB.Driver.Core/MongoException.cs

```diff
@@ -57,6 +62,36 @@ public MongoException(string message, Exception innerException)
         public MongoException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
+            _errorLabels = (List<string>)info.GetValue(nameof(_errorLabels), typeof(List<String>));
+        }
+#endif
+
+        // public properties
+        /// <summary>
+        /// Gets the error labels.
+        /// </summary>
+        /// <value>
+        /// The error labels.
+        /// </value>
+        public IReadOnlyList<string> ErrorLabels => _errorLabels;
```

Comment:
For consistency with the spec/other drivers, add a HasErrorLabel method as well.

Reply:
Done. I didn't originally add it because the following seemed to cover it:

    subject.ErrorLabels.Contains(someLabel);